### PR TITLE
Update renew-the-federation-certificate-exchange-2013-help.md

### PR DESCRIPTION
**Pull request by andreig2735**
The fact that the old expired certificate is still referenced in the federation trust object is causing confusion among exchange administrators since they want to remove the expired certificate. This issue is also a call generator in CSS.

For more clarity please see this issue:
https://github.com/MicrosoftDocs/OfficeDocs-Exchange/issues/1711 

and the blog post comments here by Mirela_Buru 

https://techcommunity.microsoft.com/t5/exchange-team-blog/keep-your-federation-trust-up-to-date/ba-p/2088788

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeServer2013/renew-the-federation-certificate-exchange-2013-help.md [hunk 1]
```diff
@@ -21,6 +21,8 @@ This topic explains how to update the self-signed federation certificate that's
 
 - If the federation certificate has already expired, follow the steps in the Replace an expired federation certificate section.
 
+**Note:** by design, after renewing the certificate, the expired certificate associated with the Federation trust cannot be removed from the Federation trust object.
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeServer2013/renew-the-federation-certificate-exchange-2013-help.md:24]:
```suggestion
    > [NOTE]
    > By design, after renewing the certificate, the expired certificate associated with the federation trust cannot be removed from the federation trust object. 
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @andreig2735, kindly commit the suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by andreig2735** [comment]
Is there anything else needed ? I still can't see the approved changes in the public article. Thanks

**Pull request merged**